.textlintrc.json

```diff
@@ -126,6 +127,8 @@
         "Home Assistant Supervised",
         "Home Assistant Supervisor",
         "Home Assistant",
+        "Home Assistant SkyConnect",
```

Comment:
Maybe add `SkyConnect` as a single word too?

Reply:
Done. Thank you for the hint!